spec/requests/private_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PrivateController, type: :controller do
+  describe 'GET#index' do
+    context 'when user is not sign in' do
+      it 'returns redirect status' do
+        get :index
+
+        expect(response).to have_http_status(:found)
+      end
+    end
+
+    context 'when user is sign in' do
+      before(:each) do
+        sign_in Fabricate(:user)
+      end
+
+      it 'returns expected status' do
```

Comment:
```suggestion
      it 'returns success status' do
```

Let's try to avoid `expected`, `correct`, `calculated` ... words in a test description. Tests should "document" the code and so when reading the test description, a developer should understand exactly what the code is expect to do.

E.x. "given 2 coffee at 1.5USD each", "it returns the price of 3USD"
=> By just reading these 2 simple descriptions, you have a clear understanding that the method calculate the price addition of a list of coffee based on their price. ;-)
If I just say "it returns the correct price", the calculation becomes "implicit" and you have to read the test implementation to understand the code.